src/View/View.php

```diff
@@ -1259,7 +1259,7 @@ protected function _paths($plugin = null, $cached = true)
             $themePaths,
             $pluginPaths,
             $templatePaths,
-            [dirname(__DIR__) . DS . 'Template' . DS]
+            [CAKE . 'Template' . DS]
```

Comment:
This constant is not defined in CakePHP. Is there another path we can use that doesn't rely on external constants?